frontend/class-frontend.php

```diff
@@ -1334,7 +1334,16 @@ public function noindex_page() {
 	 */
 	public function noindex_feed() {
 
-		if ( ( is_feed() || is_robots() ) && headers_sent() === false ) {
+		/**
+		 * Allow switching the X-Robots-Tag header for feeds on or off.
+		 *
+		 * @api boolean $noindex Whether the noindex header should get set (true) or not (false). Defaults to true.
+		 *
+		 * @since 11.6
+		 */
+		$noindex = apply_filters( 'wpseo_feed_noindex', true );
+
+		if ( ( is_feed() || is_robots() ) && headers_sent() === false && $noindex === true ) {
```

Comment:
Please move the `$noindex` to be first in the comparison so the others don't have to be executed if we already know we're not doing this.

Reply:
Done, also moved the `headers_sent` as the second check because I thought that would also take less computing power than the `||` check with 2 WordPress functions. Please let me know if that isn't the right thing to do.